data/Templates/FhirToHL7v2/Mutators/_PatientIdentifiersToCX.liquid

```diff
@@ -0,0 +1,70 @@
+{%comment%}
+{%endcomment%}
+{%- assign acceptedCodes = "MR, SS" | split: ", " -%}
+{%- capture acceptedSystemsStr -%}
+http://example.com/v2-to-fhir-converter/assigning-authority-universal-system-HCD,
+http://example.com/v2-to-fhir-converter/assigning-authority-local-system-USSSA,
+{%- endcapture -%}
+{%- assign acceptedSystems = acceptedSystemsStr | strip_newlines | split: "," -%}
+{%-comment-%} skip the first item that is sourced from PID-2 {%-endcomment-%}
```

Comment:
I was curious if there might be another way to exclude this item. After looking at the object it seems there might be, through the organization, which leads me to wonder if we're going about this backwards.

I wonder if there's (we would probably know if we were actually solving a problem here) always "official" or "most authority" organizations that could help us locate the Identifiers/objects that are most "interesting" for PID.

🤔

Reply:
I have some notes on this. I was trying to figure out the most succinct filter I could. This is the table of identifiers from the expected JSON file. 

| Name    | Value           | Code    | Assigner        | System |
| ------- | --------------- | ------- | --------------- | ------------ |
| PID-2   | 1234567         | MR      |                 | local/test |
| PID-3   | PATID1234       | MR      | Org/b3ad        | unvsl/HCD |
| PID-3   | 123456789       | SS      |                 | local/USSSA |
| PID-4   | PATID567        |         |                 | local/test2 |
| PID-19  | PSSN123121234   | SB      |                 | http://hl7.org/fhir/sid/us-ssn |
| PID-20  | DLN-123         | DL      |                 | http://example.com/v2-to-fhir-converter/CodeSystem/US |
| GT1-2   | 1516            | MR      |                 | local/test |
| GT1-19  | 55121           | FI      |                 |            |
| DB1-3   | DB123           | MR      | Org/4e68        | urn:oid:1.12 |
| IN2-6   | Medicare-12345  | MC      |                 | http://hl7.org/fhir/sid/us-medicare |
| IN2-8   | MCN-008         | MA      |                 | http://example.com/v2-to-fhir-converter/identifier-medicaid |
| IN2-10  | MI-12345        | MI      |                 |  |
| IN2-61  | PNM1234         | MR      | Org/1fca        | unvsl/HCD |

Which is why I landed on filtering by MR, SS and also HCD to try and eliminate GT1-2, DB1-3, etc.

Reply:
Then do we even need that slice to get rid of the first element? 🤔